tracee-ebpf/tracee/tracee.go

```diff
@@ -134,8 +134,13 @@ type netProbe struct {
 
 // Validate does static validation of the configuration
 func (cfg OutputConfig) Validate() error {
-	if cfg.Format != "table" && cfg.Format != "table-verbose" && cfg.Format != "json" && cfg.Format != "gob" && !strings.HasPrefix(cfg.Format, "gotemplate=") {
-		return fmt.Errorf("unrecognized output format: %s. Valid format values: 'table', 'table-verbose', 'json', 'gob' or 'gotemplate='. Use '--output help' for more info.", cfg.Format)
+	if cfg.Format != "table" &&
+		cfg.Format != "table-verbose" &&
+		cfg.Format != "json" &&
+		cfg.Format != "gob" &&
+		cfg.Format != "ignore" && // TODO: add to test after merging https://github.com/aquasecurity/tracee/pull/881
```

Comment:
We try to avoid TODO's in comments, better to just make a github issue

Reply:
@grantseltzer okay, but the PR was just merged, I’ll rebase and resolve the TODO. Thanks